### PR TITLE
[docs] Add APM to simplified security docs

### PR DESCRIPTION
**Pull request by bmorelli25**
* Persists the APM xpack security changes in https://github.com/elastic/apm-server/pull/2228. Waiting for approval on the APM side before marking this as ready for review.
* Follow up to DeDe's work on https://github.com/elastic/beats/pull/11329.

### CODE REVIEW

File: libbeat/docs/monitoring/monitoring-beats.asciidoc [hunk 1]
```diff
@@ -40,7 +40,7 @@ the same {es} cluster, specify the following minimal configuration:
 --
 ["source","yml",subs="attributes"]
 --------------------
-xpack.monitoring:
+monitoring:
```
Comment (bmorelli25) [inline libbeat/docs/monitoring/monitoring-beats.asciidoc]: Should this be just be `monitoring`?

Comment (dedemorton) [reply libbeat/docs/monitoring/monitoring-beats.asciidoc]: yes, for 7.2 and later. If you decide to backport to 7.1 or earlier, you'll need to change the name as needed.

File: libbeat/docs/security/users.asciidoc [hunk 2]
```diff
@@ -116,13 +123,28 @@ endif::[]
 unless you've disabled automatic template loading
 
 .2+|Send data to a secured cluster that supports index lifecycle management
-|`manage_index_templates`,`manage_ilm` footnote:[Use `read_ilm` instead of
+|`manage_index_templates`, `manage_ilm` footnote:[Use `read_ilm` instead of
```
Comment (dedemorton) [inline libbeat/docs/security/users.asciidoc]: good eye!

**Comment by bmorelli25** [comment]
In testing I found a build error with heartbeat: they use the `no_dashboards` attribute which was causing asciidoctor to attempt to build a blank "Privileges needed by {kib} users" table (it doesn't like that). Updated the attribute to only rule out APM Server.

**Pull request merged**